conda_forge_tick/cli.py

```diff
@@ -0,0 +1,169 @@
+import argparse
+import time
+import os
+import subprocess
+
+from doctr.travis import run_command_hiding_token as doctr_run
+
+from .utils import load_graph
+
+from .all_feedstocks import main as main_all_feedstocks
+from .make_graph import main as main_make_graph
+from .update_upstream_versions import main as main_update_upstream_versions
+from .auto_tick import main as main_auto_tick
+from .status_report import main as main_status_report
+from .audit import main as main_audit
+from .update_prs import main as main_update_prs
+from .mappings import main as main_mappings
+
+INT_SCRIPT_DICT = {
+    0: main_all_feedstocks,
+    1: main_make_graph,
+    2: main_update_upstream_versions,
+    3: main_auto_tick,
+    4: main_status_report,
+    5: main_audit,
+    6: main_update_prs,
+    7: main_mappings,
+}
+
+
+def _run_git_cmd(cmd):
+    return subprocess.run(cmd, shell=True, check=True)
+
+
+def deploy(args):
+    """Deploy the graph to github"""
+    if args.dry_run:
+        print("(dry run) deploying")
+        return
+
+    CIRCLE_BUILD_URL = os.environ.get("CIRCLE_BUILD_URL", "")
+    for cmd in [
```

Comment:
might as well collapse a bunch of the `git add`s with a list comp?

Reply:
IDK if the code is clearer now or not but here you go!